srcpkgs/xsel/template

```diff
@@ -10,3 +10,4 @@ license="custom"
 homepage="http://www.vergenet.net/~conrad/software/xsel/"
 distfiles="http://www.vergenet.net/~conrad/software/xsel/download/xsel-${version}.tar.gz"
 checksum=b927ce08dc82f4c30140223959b90cf65e1076f000ce95e520419ec32f5b141c
+alternatives="xsel:xsel:/usr/bin/xsel"
```

Comment:
This is request to create symlink /usr/bin/xsel pointing to itself. Binary needs to be renamed, see `tealdeer` template.